tokio/src/net.rs

```diff
@@ -65,6 +65,12 @@ pub mod signal {
     pub use tokio_net::signal::ctrl_c;
 }
 
+#[cfg(feature = "process")]
+pub mod process {
+    //! An implementation of asynchronous process management for Tokio.
+    pub use tokio_net::process::*;
```

Comment:
We tend not to use glob re-exports to avoid accidentally re-exporting too much if something was mistakenly marked as `pub`.

```suggestion
    pub use tokio_net::process::{Child, ChildStderr, ChildStdin, ChildStdout, Command};
```

Reply:
Changed.